src/ibmca_rsa.c

```diff
@@ -264,7 +265,7 @@ static int ibmca_mod_exp_crt(BIGNUM * r, const BIGNUM * a,
 
     rc = p_ica_rsa_crt(ibmca_handle, input, key, output);
     if (rc != 0) {
-        IBMCAerr(IBMCA_F_IBMCA_MOD_EXP, IBMCA_R_REQUEST_FAILED);
+        //IBMCAerr(IBMCA_F_IBMCA_MOD_EXP, IBMCA_R_REQUEST_FAILED);
```

Comment:
why ?

Reply:
Because now this function fails if the request is too big for the card.  So we would add the error to OpenSSL and then use a fallback to successfully compute the modular exponentiation via a software fallback.  That seemed wrong to me.  However, I did not want to remove the line just yet since we might find an even better solution to this problem at some point.  Anyway, I found it weird to get lots of errors from the openssl speed test for RSA even though there were no actual errors.